ccdproc/image_collection.py

```diff
@@ -58,14 +58,19 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     ext: str or int, optional
+         Name/Index of the extension, which specifies the extension to extract data
```

Comment:
The first part contains some redundancy "[...]the extension which specifies the extension[...]". Also "to extract fata from all files in a given directory" is not really specific for the parameter. The `ext` parameter specifies the extension from which to extract the header/data, right?